TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/workflow/commands/output.py

```diff
@@ -0,0 +1,277 @@
+                f"The pod may have been cleaned up or not yet created.",
+                err=True
+            )
+        else:
+            click.echo(f"\nError retrieving output: {e}", err=True)
+        ctx.exit(ExitCode.FAILURE.value)
+
+
+@click.command(name="output")
+@click.argument('workflow_name', required=False)
+@click.option(
+    '--argo-server',
+    default=f"http://{os.environ.get('ARGO_SERVER_SERVICE_HOST', 'localhost')}"
+    f":{os.environ.get('ARGO_SERVER_SERVICE_PORT', '2746')}",
+    help='Argo Server URL (default: ARGO_SERVER env var, or ARGO_SERVER_SERVICE_HOST:ARGO_SERVER_SERVICE_PORT)'
+)
+@click.option(
+    '--namespace',
+    default='ma',
+    help='Kubernetes namespace for the workflow (default: ma)'
+)
+@click.option(
+    '--insecure',
+    is_flag=True,
+    default=False,
+    help='Skip TLS certificate verification'
+)
+@click.option(
+    '--token',
+    help='Bearer token for authentication'
+)
+@click.option(
+    '--tail-lines',
+    type=int,
+    default=100,
+    help='Number of lines to show from the end of the output (default: 100)'
+)
+@click.pass_context
+def output_command(ctx, workflow_name, argo_server, namespace, insecure, token, tail_lines):
+    """Show output for workflow steps.
+
+    Displays workflow steps and allows interactive selection to view output.
+    Only shows output for Pod-type steps that have executed.
+
+    Example:
+        workflow output
+        workflow output my-workflow
+        workflow output --tail-lines 50
+    """
+    try:
+        service = WorkflowService()
+
+        # Auto-detect workflow if name not provided
+        if not workflow_name:
+            workflow_name = auto_detect_workflow(service, namespace, argo_server, token, insecure, ctx)
+            if not workflow_name:
+                return
+
+        # Get workflow status
+        result = service.get_workflow_status(
+            workflow_name=workflow_name,
+            namespace=namespace,
+            argo_server=argo_server,
+            token=token,
+            insecure=insecure
+        )
+
+        if not result['success']:
+            click.echo(f"Error getting workflow status: {result['error']}", err=True)
+            ctx.exit(ExitCode.FAILURE.value)
+
+        # Display workflow status using existing function from status.py
+        # Don't show the output hint since we're already in the output command
+        _display_workflow_status(result, show_output_hint=False)
+
+        # Filter for Pod-type nodes from step_tree
+        step_tree = result.get('step_tree', [])
+        pod_nodes = [node for node in step_tree if node['type'] == 'Pod']
+
+        if not pod_nodes:
+            click.echo("\nNo pod steps found in this workflow.")
+            return
+
+        # Display selection menu and get user choice
+        _display_step_menu(pod_nodes)
+        choice_idx = _get_user_choice(pod_nodes, ctx)
```

Comment:
I'd like to see a command line option to do this (& all of the workflow subcommands) so that we can tie it better into integ tests